src/main/java/RedBot/Bot.java

```diff
@@ -1,11 +1,8 @@
 package RedBot;
 
 import java.lang.Character;
```

Comment:
I just noticed this import. It's unnecessary since `java.lang` is the default package in Java and explicitly importing it here is redundant.